packages/editor-ui/src/components/Node.vue

```diff
@@ -207,13 +239,35 @@ export default mixins(externalHooks, nodeBase, nodeHelpers, workflowHelpers).ext
 		shiftOutputCount (): boolean {
 			return !!(this.nodeType && this.nodeType.outputs.length > 2);
 		},
-	},
+		shouldShowTooltip () : boolean {
```

Comment:
nitpick - just can be a little more clear... since we have another tooltip in nodes (error tooltip)
```suggestion
		shouldShowTriggerTooltip () : boolean {
```

Reply:
Refactored